src/fu-util.c

```diff
@@ -356,51 +361,60 @@ fu_util_perhaps_show_unreported(FuUtilPrivate *priv, GError **error)
 	/* show the success and failures */
 	if (!priv->assume_yes && !all_automatic) {
 		/* delimit */
-		g_print("________________________________________________\n");
+		fu_console_println_literal(priv->console,
+					   "________________________________________________");
```

Comment:
This seems like something that a helper should exist for to me.  Create a line that is "X" characters long

Reply:
Good idea, fixed. I've also used the UTF-8 ─ char rather than _.